supabase/functions/admin-update-user/index.ts

```diff
@@ -0,0 +1,75 @@
+// @ts-nocheck (for Deno references, annoyingly cannot get around for now)
+/* global Deno */
+
+import { type Handler, serve } from "https://deno.land/std@0.168.0/http/server.ts";
+import { createClient } from "https://esm.sh/@supabase/supabase-js@latest";
+import { corsHeaders } from "../_shared/cors.ts";
+import { AdminUserAttributes } from "https://esm.sh/@supabase/gotrue-js@latest";
+
+interface UpdateUserRequestBody {
+    userId: string;
+    attributes: AdminUserAttributes;
+}
+
+serve(async (req: Handler): Promise<Response> => {
+    if (req.method === "OPTIONS") {
+        return new Response("ok", { headers: corsHeaders });
```

Comment:
corsHeaders? Can this be renamed or no?

Reply:
I feel like the `corsHeaders` name is sensible here

Reply:
Nah, it's gotta go under `headers`